test/rules/typedefRuleTests.ts

```diff
@@ -34,92 +34,133 @@ describe("<typedef, not enabled>", () => {
+        assertFailures(expectedFailure1, expectedFailure2, expectedFailure3, expectedFailure4, expectedFailure5);
     });
 
-    it("enforces typedef in property-signature", () => {
+    it("enforces typedef in property declaration", () => {
         var expectedFailure = Lint.Test.createFailure(fileName,
-            [10, 9],
-            [10, 10],
-            "expected property-signature: 'Prop' to have a typedef");
+            [22, 9],
+            [22, 10],
+            "expected property-declaration: 'Prop' to have a typedef");
 
         assertFailures(expectedFailure);
     });
 
-    it("enforces typedef in variable declarator", () => {
+    it("enforces typedef in variable declaration", () => {
         var expectedFailure1 = Lint.Test.createFailure(fileName,
-            [7, 2],
-            [7, 3],
-            "expected variable-declarator: 'NoTypeObjectLiteralWithPropertyGetter' to have a typedef");
+            [1, 42],
+            [1, 43],
+            "expected variable-declaration: 'NoTypeObjectLiteralWithPropertyGetter' to have a typedef");
         var expectedFailure2 = Lint.Test.createFailure(fileName,
-            [18, 2],
-            [18, 3],
-            "expected variable-declarator: 'NoTypesFn' to have a typedef");
+            [26, 14],
+            [26, 15],
+            "expected variable-declaration: 'NoTypesFn' to have a typedef");
         var expectedFailure3 = Lint.Test.createFailure(fileName,
-            [17, 18],
-            [17, 19],
-            "expected variable-declarator: 'c' to have a typedef");
+            [29, 10],
+            [29, 11],
+            "expected variable-declaration: 'c' to have a typedef");
         var expectedFailure4 = Lint.Test.createFailure(fileName,
-            [17, 18],
-            [17, 19],
-            "expected variable-declarator: 'd' to have a typedef");
-        var expectedFailure5 = Lint.Test.createFailure(fileName,
-            [23, 27],
-            [23, 28],
-            "expected member-variable-declarator: 'Prop' to have a typedef");
-        assertFailures(expectedFailure1, expectedFailure2, expectedFailure3, expectedFailure4, expectedFailure5);
+            [30, 10],
+            [30, 11],
+            "expected variable-declaration: 'd' to have a typedef");
+
+        assertFailures(expectedFailure1, expectedFailure2, expectedFailure3, expectedFailure4);
+    });
+
+    it("enforces typedef in member variable declaration", () => {
+        var expectedFailure = Lint.Test.createFailure(fileName,
+            [36, 11],
+            [36, 12],
+            "expected member-variable-declaration: 'Member' to have a typedef");
+
+        assertFailures(expectedFailure);
     });
 
     it("only has the expected failures", function() {
         if (actualFailures.length > 0) {
-            assert(false, "got these extra errors " + JSON.stringify(actualFailures, null, 2));
+            assert(false, "got " + actualFailures.length + " extra errors");
```

Comment:
`JSON.stringify` no longer works on the failures because there's a circular reference in the `Node` object structure